### PR TITLE
better instance type default for gundeck

### PR DESCRIPTION
**Pull request by jschaul**

### CODE REVIEW

File: khan-cli/src/Khan/CLI/Cluster.hs [hunk 1]
```diff
@@ -169,6 +172,7 @@ instance Options Deploy where
 
         checkFile (_trust  dTrust)  " specified by --trust must exist."
         checkFile (_policy dPolicy) " specified by --policy must exist."
+        check (dEnv == "prod" && dRole == "gundeck" && dType /= C3_Large && (not dForceGundeck)) " gundeck should use c3.large as instance type in prod. Use --instance c3.large."
```
Comment (tiago-loureiro) [inline khan-cli/src/Khan/CLI/Cluster.hs]: Let's actually use `m3.large` which has 2x memory for only 20% more cost compared to the `c3.large`

**Comment by neongreen** [comment]
I have reservations about this change (it makes Khan even more of a weird mix between a general-purpose tool and a Wire-specific tool), but I'm also not sure writing a bunch of `./deploy-{brig|galley|...}` scripts would be a better approach.

**Comment by tiago-loureiro** [comment]
> [...] but I'm also not sure writing a bunch of ./deploy-{brig|galley|...} scripts would be a better approach.

Exactly; we'll end up using khan directly and not through a script so we'll forget about it anyway.

> (it makes Khan even more of a weird mix between a general-purpose tool and a Wire-specific tool)

khan as a general purpose is long gone :) we need to phase this out, let's talk about these plans next week when we gather.

**Comment by jschaul** [comment]
> I have reservations about this change

Alternatively, deployments to prod could be handled by CI, which would then contain the right arguments in a bash script.

**Comment by fisx** [comment]
> > I have reservations about this change
> 
> Alternatively, deployments to prod could be handled by CI, which would then contain the right arguments in a bash script.

I like this better, too.

(Closing; let me know if I shouldn't have done that.)

**Pull request closed without merging**